plugins/csp-web-api/gui/example-web-frontend.html

```diff
@@ -246,9 +248,10 @@
                 </tr>
                 <tr>
                   <td>format</td>
-                  <td>png <br> tiff</td>
+                  <td>png <br> tiff <br> raw</td>
                   <td>This can be either 'png', 'jpeg', or 'tiff'. It defaults to 'png' for color
```

Comment:
```suggestion
                  <td>png <br> jpeg <br> tiff <br> raw</td>
                  <td>This can be either 'png', 'jpeg', 'tiff', or 'raw'. It defaults to 'png' for color
```